### PR TITLE
Add mathics-threejs-backend version

### PR DESCRIPTION
**Pull request by TiagoCavalcanteTrindade**
We still need to release mathics-threejs-backend 0.5.0.

### CODE REVIEW

File: mathics/builtin/box/graphics3d.py [hunk 1]
```diff
@@ -388,6 +388,7 @@ def boxes_to_json(self, leaves=None, **options):
                 },
                 "lighting": self.lighting,
                 "viewpoint": self.viewpoint,
+                "version": "0.5.0",
```
Comment (rocky) [inline mathics/builtin/box/graphics3d.py:391]: Let's go with 1.0.0

**Review by rocky — APPROVED** [review]
LGTM

Thanks for the densiityplot fix

File: mathics/builtin/box/graphics3d.py [hunk 2]
```diff
@@ -388,6 +388,7 @@ def boxes_to_json(self, leaves=None, **options):
                 },
                 "lighting": self.lighting,
                 "viewpoint": self.viewpoint,
+                "revision": "1.0",
```
Comment (rocky) [inline mathics/builtin/box/graphics3d.py:391]: I thought we were going to use "protocol" ?

**Comment by TiagoCavalcanteTrindade** [comment]
> Thanks for the densityplot fix

To say the truth that shouldn't even be in the commit, I forgot I have done that and made a `git add .` 😁, anyway, it is working the same.

File: mathics/builtin/box/graphics3d.py (hunk 2 above)
Comment (TiagoCavalcanteTrindade) [reply mathics/builtin/box/graphics3d.py:391]: My typo, I've written protocol everywhere else.

**Pull request merged**